commit 066e52618ed6f4ef25121716b95b19cb8f90a5b4
Author: Marsell Kukuljevic <marsell@joyent.com>
Date:   2019-04-10T20:38:22+00:00 (6 months ago)
    
    TRITON-1384 CNAPI/DAPI blows assertion "callback (Function) is required" when doing getVm
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/lib/algorithms/hard-filter-volumes-from.js b/lib/algorithms/hard-filter-volumes-from.js
index f8fd646..220d2d7 100644
--- a/lib/algorithms/hard-filter-volumes-from.js
+++ b/lib/algorithms/hard-filter-volumes-from.js
@@ -121,7 +121,7 @@ loadVmAndFilter(log, servers, requiredVms, getVm, cb)
 
 		var vmUuid = remainingVms.shift();
 
-		return getVm({ uuid: vmUuid }, function (err, vm) {
+		return getVm({ uuid: vmUuid }, {}, function getVmCb(err, vm) {
 			if (err) {
 				log.error('Error loading VM', err);
 
